core/codegen/src/derive/from_form.rs

```diff
@@ -244,14 +244,16 @@ pub fn derive_from_form(input: proc_macro::TokenStream) -> TokenStream {
             .try_field_map(|_, f| {
                 let (ident, ty, name_view) = (f.ident(), f.stripped_ty(), f.name_view()?);
                 let validator = validators(f, &ident, true)?;
+                let default = defaults(f)?;
```

Comment:
Let's call this `user_default` or something to that effect, to disambiguate from the form's default.